doc-content/enterprise-only/optimizer/examples-table-ref.adoc

```diff
@@ -0,0 +1,227 @@
+[id='examples-table-ref']
+= Table of examples
```

Comment:
A few tweaks below, plus IBM/RH style guides don't like the type of bulleted list there, prefer something like I've shown:

```
= Table of {PLANNER} examples

Some of the {PLANNER} examples ... for the purpose **[note typo]** of a contest.

A realistic contest is an official, independent contest with the following standards:

* Clearly defined real-world use cases
* Real-world constraints
* Reproducible results ...
* Serious participation from ...

Realistic contests provide ...

.Examples overview [sentence style cap]
[Consider re-ordering either the table or the sections that follow so that they're in a matching order. Odd that they don't correlate.]
```